### PR TITLE
Backports 0.17 pr25

### PR DESCRIPTION
**Pull request by PastaPastaPasta**
https://github.com/dashpay/dash/commit/7776fd8937809ed1e4d63d115c08c2b5fee78970 is causing a test failure that I tried fixing, however, wasn't able to fully fix. LMK

### CODE REVIEW

File: test/functional/interface_zmq.py [hunk 1]
```diff
@@ -3,17 +3,17 @@
...
 
+from test_framework.test_framework import (
+    BitcoinTestFramework, skip_if_no_bitcoind_zmq, skip_if_no_py3_zmq)
 from test_framework.mininode import dashhash
-from test_framework.test_framework import BitcoinTestFramework, SkipTest
 from test_framework.util import (assert_equal,
                                  bytes_to_hex_str,
                                  hash256,
-                                )
+                                 )
```
Comment (xdustinface) [inline test/functional/interface_zmq.py:16]: I guess this was not intentionally?

**Review by xdustinface — CHANGES_REQUESTED** [review]
See one review comment + the following

e464ae0854cbce54ff6df741e4b323caca2487fa Fixes the util test issue 

ec22d07c4abc5db3334ac56d1d682077c5c4f49d Fixes the python linter. (300572e4f1d1af22f0fad125564d1e72e3dae88e introduced `CMerkleBlock` and `CPartialMerkleTree` which we both had already obviously because of `feature_dip4_coinbasemerkleroots.py`)

bc8cf7494b79884437381515bf202afa96969f63 is doing something similar like 15d6516c42adc06eba34e9afb50997d67438ed17 in our dash zmq tests.

ba9b7265bbd0fb00559a6e97f91cfd626da06eba Just dashify

File: test/functional/interface_zmq.py (hunk 1 above)
Comment (PastaPastaPasta) [reply test/functional/interface_zmq.py:16]:
```suggestion
                                )
```

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
Pls see https://github.com/UdjinM6/dash/commits/pr3882

**Review by UdjinM6 — APPROVED** [review]
LGTM 👍 

utACK (tests pass locally)

**Review by xdustinface — APPROVED** [review]
utACK

**Pull request merged**